js/app.js

```diff
@@ -616,6 +645,30 @@
 
 			OCA.SpreedMe.app.audioDisabled = true;
 		},
+		availableVideo: function() {
+			var $hideVideoButton = $('#hideVideo');
+
+			navigator.getUserMedia = navigator.getUserMedia ||
+                         navigator.webkitGetUserMedia ||
+                         navigator.mozGetUserMedia;
+
+			if (navigator.getUserMedia !== null) {
+				navigator.getUserMedia({ video: true },
```

Comment:
There should be no need to call `getUserMedia` again, this was already done inside `SimpleWebRTC`. Also the streams here could be different from what was returned to the original `getUserMedia`.

Reply:
you are right!